docs-v2/pages/code/python/index.mdx

```diff
@@ -126,6 +126,10 @@ Unlike the [Node.js equivalent](https://pipedream.com/docs/workflows/steps/trigg
 _Don't forget_ to [configure your workflow's HTTP trigger to allow a custom response](/workflows/steps/triggers/#http-responses). Otherwise your workflow will return the default response.
 </Callout>
 
+## Invoke another workflow
+
+This is currently not supported in Python steps. However, you can invoke another workflow using Node.js steps. For more information, see [Invoke another workflow in Node.js](/code/nodejs#invoke-another-workflow).
```

Comment:
I'd recommend documenting the HTTP trigger flow, since you can still call a workflow with that. Then maybe just remove the reference to the Node.js stuff for now.

Reply:
I actually removed it from the Python docs and added it to the Troubleshooting docs. Also added the built-in action reference